src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-q_a.js

```diff
@@ -1,13 +1,22 @@
-const contentModelType = 'paragraph-q_a';
-
 module.exports = {
   type: 'object',
   properties: {
-    baseType: {
-      enum: ['paragraph'],
-    },
-    contentModelType: {
-      enum: [contentModelType],
+    // entityId: { type: 'string' }, // Also should validate that the string is a number
```

Comment:
Is this commented out because the transformed entity is supposed to have it, but it doesn't yet because of tome-sync?

Reply:
Exactly, yeah.